std/variant.d

```diff
@@ -1969,19 +1969,19 @@ unittest
 }
 
 /**
- * Applies a delegate or function to the given Algebraic depending on the held type,
+ * Applies a delegate or function to the given $(LREF Algebraic) depending on the held type,
  * ensuring that all types are handled by the visiting functions.
  *
  * The delegate or function having the currently held value as parameter is called
  * with $(D variant)'s current value. Visiting handlers are passed
  * in the template parameter list.
- * It is statically ensured that all types of
+ * It is statically ensured that all held types of
  * $(D variant) are handled across all handlers.
  * $(D visit) allows delegates and static functions to be passed
  * as parameters.
  *
  * If a function without parameters is specified, this function is called
- * when variant doesn't hold a value. Exactly one parameter-less function
+ * when `variant` doesn't hold a value. Exactly one parameter-less function
```

Comment:
`variant` (in quotes) refers to the runtime argument of `visit` - the existing docs use this style, e.g. on line 1979 - only here they forgot to add the `$(D )` quotes.